solidity/contracts/liquidity-protection/LiquidityProtectionStats.sol

```diff
@@ -55,7 +55,7 @@ contract LiquidityProtectionStats is ILiquidityProtectionStats, AccessControl {
 
     /**
      * @dev increases the total amounts
-     * can only be executed only by an owner
+     * can be executed only by an owner
```

Comment:
previous version is cleaner

Reply:
Previous version is wrong (an extra 'only').